client/src/components/SignUp.style.js

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
+  background-color: #f9d52f;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const Title = styled.h1`
+  fontfamily: Work Sans;
+  fontstyle: Black;
+  fontsize: 38px;
+  background: white;
+  margin: 50px;
+`;
+export const StyledForm = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+export const InputContainer = styled.div``;
+export const StyledLabel = styled.label`
+  font-size: 18px;
```

Comment:
please use ems, and color variables.
Also these labels might need a font family